### ISSUE TITLE
Demo application doesn't build out of the box

### ISSUE THREAD

**Issue by peerst** [grisp/grisp2-rtems-toolchain#5]
Seems not to be able to find the cross compiler:

```
yrael:~/projects/grisp/grisp-2/grisp2-rtems-toolchain/demo $ make
mkdir b-imx7
arm-rtems5-gcc --pipe  -MT b-imx7/i2c.o -MD -MP -MF b-imx7/i2c.d -B /Users/peer/projects/grisp/grisp-2/grisp2-rtems-toolchain/demo//../rtems/5/arm-rtems5/imx7/lib -specs bsp_specs -qrtems -Wall -Wextra -Wconversion -Wformat-security -Wformat=2 -Wshadow -Wcast-qual -Wcast-align -Wredundant-decls -Wstrict-prototypes -Wbad-function-cast -march=armv7-a -mthumb -mfpu=neon -mfloat-abi=hard -mtune=cortex-a7 -O2 -g -ffunction-sections -fdata-sections -O0 -c i2c.c -o b-imx7/i2c.o
make: arm-rtems5-gcc: No such file or directory
```

One would assume looking at the Makefile that the compiler should be configured in the included mk file but apparently it doesn't

**Comment by c-mauderer** [comment]
I have built it using the `make demo` from the root directory most of the time. I set the PATH there to include the binaries. But you are right: I would have expected that it works directly too. I'll try to take a look at it somewhen during next week.

**Comment by peerst** [comment]
Ah didn't know about `make demo` apparently there is more problems see #6

**Comment by peerst** [comment]
But I should have RTFM ... its mentioned in the README

**Comment by eproxus** [comment]
Is this still an issue?

**Comment by c-mauderer** [comment]
From my point of view it should work. But there is a good workaround. So it currently has a very low priority for me except you need it.

**Comment by eproxus** [comment]
@peerst ?

**Comment by eproxus** [comment]
This is fixed in #3 (11286f03ad9c92d57d0eb743c7e33ff8b2bbf288).
